cdap-ui/app/cdap/components/SortableStickyTable/index.js

```diff
@@ -27,9 +27,9 @@ export default class SortableStickyTable extends SortableTable {
       <thead>
         <tr>
           {
-            this.props.tableHeaders.map(tableHeader => {
+            this.props.tableHeaders.map((tableHeader, i) => {
               return (
-                <th>
+                <th key={i}>
```

Comment:
should this key be the value of the tableHeader or uniqueId? the content of the header array can change but index remains the same?

Reply:
Not sure I get the premise(Once we render the table would the header change?) but we have a usecase today that has empty header for caret toggle and delete buttons at the ends of the table. Since a table header doesn't necessarily need to have a property or label I went with the index.

Reply:
The content of the header can change right? It's not immutable once the header is rendered. For instance in dataprep, if you were to remove a column, or rename the column name.